source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace IOSFeedNS
+{
+    public static class Constatns
+    {
+        public const string connectionStringPath = "source/ios/tools/IOSFeed/ConnectString.txt";
```

Comment:
>connectionStringPath [](start = 28, length = 20)

nit: Let's try to follow the csharp naming guidelines even for tool code. ([https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)) ConnectionStringPath (Pascal) -- basically consts follow fields guidelines (ie PascalCasing)